programs/speechRecognition/speechRecognition.py

```diff
@@ -112,14 +125,39 @@ def read(self,connection):
             else:
                 print("Could not found dictionary. Check file names and directories.")
                 bottleOut.addString("Could not find dictionary. Check file names and directories.")
+        
+            
+        # Mute/unmute the mixer        
+        elif bottleIn.get(0).asString() == "setMic" and bottleIn.size() == 2:
+            # Mixer settings
+            try:
+                mixer = alsaaudio.Mixer("Capture") # Capture / Master
+            except alsaaudio.ALSAAudioError:
+                print("No such mixer: %s" %sys.stderr)
+                sys.exit(1)
+    
+            channel = alsaaudio.MIXER_CHANNEL_ALL            
+            # mute/unmute
+            if(bottleIn.get(1).asString() ==  "mute"):
+                mixer.setrec(0, channel)
+                bottleOut.addString("microphone muted")
+		print("mic muted")
```

Comment:
Tabs instead of spaces.

Reply:
By the way, if you are going to fix those whitespaces, it's worth noting that parens are not necessary around `if` clauses in Python: `if bottleIn.get(1).asString() ==  "mute":`.